german-iso.lbx

```diff
@@ -0,0 +1,16 @@
+\ProvidesFile{german-iso.lbx}
+[2016/05/15 v0.3.1 biblatex localization]
+
+\InheritBibliographyExtras{german}
+\InheritBibliographyStrings{german}
+
+\DeclareBibliographyStrings{%
+  at           = {{in}% maybe 'bei'
+                  {in}},
+  bysupervisor = {{betreut von}% maybe Betreuer, but gender-dependent then, female: Betreuerin
+                  {betreut von}},
```

Comment:
Exactly, there is no explicit section about academic theses in the standard, it should be read between the lines..